assets/stylesheets/common/discourse-reactions.scss

```diff
@@ -340,3 +325,8 @@ nav.post-controls .actions button {
     height: 15px;
   }
 }
+
+nav.post-controls .actions button:focus {
```

Comment:
can we have something that impacts only reactions button here ?

Reply:
sure, ill make it more specific

Reply:
Added a commit with changes